--- ./po/LINGUAS.org	2008-10-28 01:07:20.019448892 +0100
+++ ./po/LINGUAS	2008-10-28 01:07:30.968832590 +0100
@@ -1,2 +1,2 @@
 # set of available languages (in alphabetic order)
-am ar az be bn_IN ca cs da de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu id it ja ka ko ku lt lv mk mr ms nl nb_NO pa pl pt_BR pt_PT ro ru si sk sv ta tr uk ur vi zh_CN zh_TW
+am ar az be bn_IN ca cs da de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu id it ja ka ko ku lt lv mk mr ms nl nb pa pl pt_BR pt ro ru si sk sv ta tr uk ur vi zh_CN zh_TW
--- ./configure.in.org	2008-10-28 01:07:13.008834078 +0100
+++ ./configure.in	2008-10-28 01:07:42.808831117 +0100
@@ -66,7 +66,7 @@ AC_CHECK_FUNCS([_IceTransNoListen])
 LIBS="$ac_LIBS"
 
 dnl Check for i18n support
-XDT_I18N([am ar az be bn_IN ca cs da de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu id it ja ka ko ku lt lv mk mr ms nl nb_NO pa pl pt_BR pt_PT ro ru si sk sv ta tr uk ur vi zh_CN zh_TW])
+XDT_I18N([am ar az be bn_IN ca cs da de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu id it ja ka ko ku lt lv mk mr ms nl nb pa pl pt_BR pt ro ru si sk sv ta tr uk ur vi zh_CN zh_TW])
 
 dnl Check for required packages
 XDT_CHECK_PACKAGE([LIBXFCE4UTIL], [libxfce4util-1.0], [4.4.0])
