diff -urN xfce4-session-4.6.0/icons/48x48/Makefile.am xfce4-session-4.6.0.new/icons/48x48/Makefile.am
--- xfce4-session-4.6.0/icons/48x48/Makefile.am	2009-01-12 22:11:40.000000000 +0100
+++ xfce4-session-4.6.0.new/icons/48x48/Makefile.am	2009-02-27 23:45:12.000000000 +0100
@@ -4,9 +4,11 @@
 	xfce4-session.png						\
 	xfce4-splash.png						\
 	xfsm-hibernate.png						\
+	xfsm-lockscreen.png						\
 	xfsm-logout.png							\
 	xfsm-reboot.png							\
 	xfsm-shutdown.png						\
-	xfsm-suspend.png
+	xfsm-suspend.png						\
+	xfsm-switch.png
 
 EXTRA_DIST = $(icons_DATA)
