--- xfce4-session-4.7.1/xfce4-session/xfsm-splash-screen.c~	2010-11-03 21:11:37.000000000 +0100
+++ xfce4-session-4.7.1/xfce4-session/xfsm-splash-screen.c	2010-11-15 13:08:21.252465452 +0100
@@ -280,7 +280,7 @@
   void (*init) (XfsmSplashEngine *engine);
   gchar *filename;
 
-  filename = g_module_build_path (LIBDIR "/xfce4/splash/engines", engine);
+  filename = g_module_build_path (LIBDIR "/xfce4/session/splash-engines", engine);
   splash->module = g_module_open (filename, G_MODULE_BIND_LOCAL);
   g_free (filename);
 
