diff -urN xfce4-session-4.4.0/configure.in xfce4-session-4.4.0.new/configure.in
--- xfce4-session-4.4.0/configure.in	2007-01-22 17:56:48.000000000 +0100
+++ xfce4-session-4.4.0.new/configure.in	2007-01-22 17:58:45.000000000 +0100
@@ -66,7 +66,7 @@
 LIBS="$ac_LIBS"
 
 dnl Check for i18n support
-XDT_I18N([ar az be bn_IN ca cs de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu it ja ka ko lt mk mr ms nl nb_NO pa pl pt_BR pt_PT ro ru sv sk ta tr uk vi zh_CN zh_TW])
+XDT_I18N([ar az be bn_IN ca cs de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu it ja ka ko lt mk mr ms nl nb pa pl pt_BR pt ro ru sv sk ta tr uk vi zh_CN zh_TW])
 
 dnl Check for required packages
 XDT_CHECK_PACKAGE([LIBXFCE4UTIL], [libxfce4util-1.0], [4.4.0])
